### PR TITLE
Merge CDT Cloud update to 1.45.1

### PR DESCRIPTION
**Pull request by ndoschek**
#### What it does

- Updates CDT cloud Blueprint to Theia Blueprint 1.45.1 (f037dcc)
- Updates theia-traceviewer to 0.2.0

Contributed on behalf of STMicroelectronics

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

Part of https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/issues/83

#### How to test

- Test electron application (packaged), browser application and docker application.
- Also run E2E tests (`yarn electron package:preview && yarn electron test`)

### CODE REVIEW

**Comment by ndoschek** [comment]
Hi @sdirix , could you please have a look at the update PR? Thanks :)

**Comment by ndoschek** [comment]
Thank you @sdirix for your review, I'll have a look today!

File: applications/electron/electron-builder.yml [hunk 1]
```diff
@@ -65,14 +65,14 @@ nsis:
   menuCategory: true
   oneClick: false
   perMachine: false
-  installerHeaderIcon: resources/icons/WindowsLauncherIcon/TheiaBluePrint.ico
-  installerIcon: resources/icons/WindowsLauncherIcon/TheiaBluePrint.ico
-  uninstallerIcon: resources/icons/WindowsLauncherIcon/TheiaBluePrint.ico
+  installerHeaderIcon: resources/icons/WindowsLauncherIcon/TheiaIDE.ico
+  installerIcon: resources/icons/WindowsLauncherIcon/TheiaIDE.ico
+  uninstallerIcon: resources/icons/WindowsLauncherIcon/TheiaIDE.ico
```
[the remark this answers is not in the document]
Comment (ndoschek) [reply applications/electron/electron-builder.yml:70]: Yes, the content is was not changed, the icons are still showing the CDT logo

**Comment by sdirix** [comment]
@ndoschek Can you reword the commit with the ST attribution and remove the review wording? I will then merge as is

File: theia-extensions/launcher/src/browser/create-launcher-contribution.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export class CreateLauncherCommandContribution implements FrontendApplicationCon
             if (!initialized) {
                 const messageContainer = document.createElement('div');
                 // eslint-disable-next-line max-len
-                messageContainer.textContent = nls.localizeByDefault("Would you like to install a shell command that launches the application?\nYou will be able to run the Theia IDE from the command line by typing 'theia'.");
+                messageContainer.textContent = nls.localizeByDefault("Would you like to install a shell command that launches the application?\nYou will be able to run Theia Blueprint from the command line by typing 'theia'.");
```
[the remark this answers is not in the document]
Comment (ndoschek) [reply theia-extensions/launcher/src/browser/create-launcher-contribution.ts:32]: I tested this the installation of the CLI launch command on my machine, and it is also `theia` for our CDT Cloud blueprint app.

**Comment by ndoschek** [comment]
Thanks @sdirix, I addressed your comment and updated the commit message.

**Pull request merged**